ci-integration/run-test-docker.sh

```diff
@@ -113,9 +121,11 @@ process_pid(){
                 if [[ ${FAST_FAIL} -eq 0 && -n ${CI} ]]; then
                     docker logs "${containernames[$index]}"
                 fi
-                if [[ ${FAST_FAIL} -eq 0 ]]; then
```

Comment:
The current if statement still will fail fast if the FAST_FAIL is set to false because bash treats both `true` and `false` as variables that are undefined and thus equal a length of zero. Instead, by just evaluating `FAST_FAIL` directly, we can achieve the expected behavior, i.e. when fast fail is false, don't exit cleanly. 

```bash
export FAST_FAIL=false
echo "FAST_FAIL is: $FAST_FAIL"

# Current version
if [[ ${FAST_FAIL} -eq 0 ]]; then
    echo "Current code: True. Fail fast and end the testing"
else 
    echo "Current code: False. Don't fail, keep going"
fi

# New version
if ${FAST_FAIL}; then
    echo "Updated code: True. Fail fast and end the testing"
else 
    echo "Updated code: False. Don't fail, keep going"
fi

### OUTPUT
FAST_FAIL is: false
Current code: True. Fail fast and end the testing
Updated code: False. Don't fail, keep going
```
See bash online tester link: https://rextester.com/JGGGQW47612